calyx-py/test/correctness/queues/binheap.py

```diff
@@ -1,19 +1,21 @@
 # pylint: disable=import-error
 import calyx.builder as cb
+from calyx.tuple import insert_tuplify, insert_untuplify 
 
 
 def insert_swap(prog, name, width, len, idx_w):
     """Inserts the component `swap` into the program.
 
     It takes two `idx_w`-bit inputs `a` and `b` and accepts a memory by reference.
     The memory is a `len`-element memory of `width`-bit elements.
-    It swaps the values in the memory at addresses `a` and `b`.
+    It swaps the out in the memory at addresses `a` and `b`.
     """
 
     comp = prog.component(name)
+
     a = comp.input("a", idx_w)
     b = comp.input("b", idx_w)
-    mem = comp.seq_mem_d1("mem", width, len, idx_w, is_ref=True)
+    mem = comp.comb_mem_d1("mem", width, len, idx_w, is_ref=True)
```

Comment:
Thanks for swapping us over!

Reply:
Sorry, I'll make sure to go back to `seq_mem`s soon. I originally made this change because `invoke` and `seq_mem` didn't play nicely with `icarus-verilog` ([issue](https://github.com/calyxir/calyx/issues/2117))

Reply:
Oh whoops I got turned around when I made my original comment. I thought you had changed from comb to seq, but in fact you changed the other way in order to make progress. My comment probably read super sarcastic hhahaha. Anyway yes please go back to seq and we'll be all set!